usaspending_api/database_scripts/matview_generator/mv_agency_autocomplete.json

```diff
@@ -87,7 +87,7 @@
     "-- of them has been cited as an awarding agency, we need to ensure the final outcome is flagged as ",
     "-- an awarding agency). ",
     "select ",
-    "    min(agency_id) as agency_id, ",
+    "    min(agency_id) as agency_autocomplete_id, ",
```

Comment:
While derived from the agency_id, I didn't want anyone to think this was actual, legitimate agency data.  It is a derivation and is strictly designed to work with the autocomplete.  I did this with the TAS matview for the same reason but it slipped my mind when I created this matview.  As far as I can tell, nothing has latched onto the `agency_id` yet so this should affect nothing.